filter.js

```diff
@@ -132,39 +131,39 @@ const sentEventClick = (eventName) => {
         renderElement(eventList);
       }
     };
-    const generatePrice = (e) => {
-      let price = e.cost;
-      if (Number.isFinite(price)) {
-        price = e.cost.toFixed(2).replace(/\./g, ",");
-        price += " zł";
-        return price;
+
+    const displayPriceWithCurrency = (singleEvent) => {
+      if (Number.isFinite(singleEvent.cost)) {
+        return `${singleEvent.cost.toFixed(2).replace(/\./g, ",")} zł`;
       } else {
-        return price;
+        return singleEvent.cost;
       }
     };
 
-    const displayDuration = (e) => {
-      let arrayDuration = e.duration.split(":");
-      let arrayInput = `${arrayDuration[0]} h ${arrayDuration[1]} min `;
-      return arrayInput;
+    const formatDuration = (singleEvent) => {
+      let duration = singleEvent.duration.split(":");
+
+      return `${duration[0]} h ${duration[1]} min`;
+    };
+
+    const renderEmpty = () => {
+      contenerList = `
+      <div class="lecture">
+        Ooppps. Wygląda na to, ze nie mamy nic z tym filtrowaniem. Wyślij nam zgłoszenie na <a href="mailto:hello@deisgnways.io">hello@designways.io</a>
+      </div>`;
     };
+
     const renderElement = (arrayList) => {
       let contenerList = ``;
```

Comment:
przemyśl czy nie da się pozbyć tego `contenerList` 
da się to ogarnąć bez dodatkowej zmiennej robiąc map + join() 

A jak nie to chociaż literówka bo `container`